### PR TITLE
Doc: Raft retry join

### PR DESCRIPTION
**Pull request by vishalnayak**

### CODE REVIEW

**Comment by briankassouf** [comment]
@vishalnayak can you resolve the merge conflicts?

File: website/pages/docs/configuration/storage/raft.mdx [hunk 1]
```diff
@@ -85,4 +85,49 @@ set `ha_enabled = true`.
...
+
+ ### `retry_join` stanza
+
+- `leader_api_addr` `(string: "")` - Address of a possible leader node
+
+- `leader_ca_cert` `(string: "")` - CA cert of the possible leader node
+
+- `leader_client_cert` `(string: "")` - Client certificate for the follower node to establish client authentication with the possible leader node
+
+- `leader_client_key` `(string:"")` - Client key for the follower node to establish client authentication with the possible leader node
```
Comment (briankassouf) [inline website/pages/docs/configuration/storage/raft.mdx]:
```suggestion
- `leader_client_key` `(string: "")` - Client key for the follower node to establish client authentication with the possible leader node
```

File: website/pages/docs/configuration/storage/raft.mdx [hunk 2]
```diff
@@ -85,4 +85,49 @@ set `ha_enabled = true`.
...
+- `retry_join` `(list: [])` - There can be one or more `retry_join` stanzas.
+When the raft cluster is getting bootstrapped, if the connection details of all
+the nodes are known beforehand, then specifying this config stanzas enables the
+nodes to automatically form a raft cluster. All the nodes would mention all
+other nodes that they could join using this config. When one of the nodes is
+initialized, it becomes the leader and all the other nodes will join the
+initialized node to form the cluster. See the section below that describes the
+parameters accepted by the `retry_join` stanza.
+
+ ### `retry_join` stanza
```
Comment (briankassouf) [inline website/pages/docs/configuration/storage/raft.mdx]:
```suggestion
### `retry_join` stanza
```

File: website/pages/docs/configuration/storage/raft.mdx [hunk 3]
```diff
@@ -85,4 +85,49 @@ set `ha_enabled = true`.
...
   is increased significantly as more logs will need to be replayed.
 
+- `retry_join` `(list: [])` - There can be one or more `retry_join` stanzas.
+When the raft cluster is getting bootstrapped, if the connection details of all
+the nodes are known beforehand, then specifying this config stanzas enables the
+nodes to automatically form a raft cluster. All the nodes would mention all
+other nodes that they could join using this config. When one of the nodes is
+initialized, it becomes the leader and all the other nodes will join the
+initialized node to form the cluster. See the section below that describes the
+parameters accepted by the `retry_join` stanza.
```
Comment (briankassouf) [inline website/pages/docs/configuration/storage/raft.mdx]: Could you add a few spaces to these to they line up like the fields above?

File: website/pages/docs/configuration/storage/raft.mdx [hunk 4]
```diff
@@ -85,4 +85,49 @@ set `ha_enabled = true`.
   snapshot. Servers may take longer to recover from crashes or failover if this
   is increased significantly as more logs will need to be replayed.
 
+- `retry_join` `(list: [])` - There can be one or more `retry_join` stanzas.
+When the raft cluster is getting bootstrapped, if the connection details of all
+the nodes are known beforehand, then specifying this config stanzas enables the
+nodes to automatically form a raft cluster. All the nodes would mention all
+other nodes that they could join using this config. When one of the nodes is
+initialized, it becomes the leader and all the other nodes will join the
+initialized node to form the cluster. See the section below that describes the
```
Comment (briankassouf) [inline website/pages/docs/configuration/storage/raft.mdx]:
```suggestion
leader node to form the cluster. See the section below that describes the
```

File: website/pages/docs/configuration/storage/raft.mdx [hunk 5]
```diff
@@ -85,4 +85,49 @@ set `ha_enabled = true`.
   snapshot. Servers may take longer to recover from crashes or failover if this
   is increased significantly as more logs will need to be replayed.
 
+- `retry_join` `(list: [])` - There can be one or more `retry_join` stanzas.
+When the raft cluster is getting bootstrapped, if the connection details of all
+the nodes are known beforehand, then specifying this config stanzas enables the
+nodes to automatically form a raft cluster. All the nodes would mention all
```
Comment (briankassouf) [inline website/pages/docs/configuration/storage/raft.mdx]:
```suggestion
nodes to automatically join a raft cluster. All the nodes would mention all
```

Comment (briankassouf) [inline website/pages/docs/configuration/storage/raft.mdx]: You may also want to mention that the node still needs to be unsealed if using shamir?

Comment (vishalnayak) [reply website/pages/docs/configuration/storage/raft.mdx]: Good point. Done.

File: website/pages/docs/configuration/storage/raft.mdx (hunk 3 above)
Comment (vishalnayak) [reply website/pages/docs/configuration/storage/raft.mdx]: Done.

**Pull request merged**